pvlib/temperature.py

```diff
@@ -182,10 +188,84 @@ def sapm_module(poa_global, temp_air, wind_speed, a, b):
        Model", SAND Report 3535, Sandia National Laboratories, Albuquerque,
        NM.
 
+    See also
+    --------
+    sapm_cell
+    sapm_cell_from_module
     '''
     return poa_global * np.exp(a + b * wind_speed) + temp_air
 
 
+def sapm_cell_from_module(module_temperature, poa_global, deltaT,
+                          irrad_ref=1000):
+    r'''
+    Calculate cell temperature from module temperature using the Sandia Array
+    Performance Model.
+
+    See [1]_ for details on the Sandia Array Performance Model.
+
+    Parameters
+    ----------
+    module_temperature : numeric
+        Temperature of back of module surface [C].
+
+    poa_global : numeric
+        Total incident irradiance [W/m^2].
+
+    deltaT : float
+        Parameter :math:`\Delta T` in :eq:`sapm2` [C].
+
+    irrad_ref : float, default 1000
+        Reference irradiance, parameter :math:`E_{0}` in
+        :eq:`sapm2` [W/m^2].
+
+    Returns
+    -------
+    numeric, values in degrees C.
+
+    Notes
+    -----
+    The model for cell temperature :math:`T_{C}` is given by Eq. 12 in [1]_.
+
+    .. math::
+       :label: sapm2
+
+       T_{C} = T_{m} + \frac{E}{E_{0}} \Delta T
+
+    The module back surface temperature :math:`T_{m}` is implemented in
+    :py:func:`~pvlib.temperature.sapm_module`.
+
+    Model parameters depend both on
+    the module construction and its mounting. Parameter sets are provided in
+    [1]_ for representative modules and mounting, and are coded for convenience
+    in ``pvlib.temperature.TEMPERATURE_MODEL_PARAMETERS``.
+
+    +---------------+----------------+-------+---------+---------------------+
+    | Module        | Mounting       | a     | b       | :math:`\Delta T [C]`|
+    +===============+================+=======+=========+=====================+
+    | glass/glass   | open rack      | -3.47 | -0.0594 | 3                   |
+    +---------------+----------------+-------+---------+---------------------+
+    | glass/glass   | close roof     | -2.98 | -0.0471 | 1                   |
+    +---------------+----------------+-------+---------+---------------------+
+    | glass/polymer | open rack      | -3.56 | -0.075  | 3                   |
+    +---------------+----------------+-------+---------+---------------------+
+    | glass/polymer | insulated back | -2.81 | -0.0455 | 0                   |
+    +---------------+----------------+-------+---------+---------------------+
```

Comment:
I don't like that this table is copy/pasted across multiple functions. I'm ok with it for now, but perhaps resolving #939 will allow us to remove it from both docstrings.